crates/astria-conductor/src/driver.rs

```diff
@@ -65,6 +70,8 @@ pub struct Driver {
 
     network: GossipNetwork,
 
+    block_verifier: Arc<BlockVerifier>,
```

Comment:
Just for clarification, this is behind an `Arc` because it's shared between the reader and the driver?

Reply:
yes exactly!